.github/template/version-branch-template.yml

```diff
@@ -13,6 +13,9 @@
 '14.3':
   engine_branch: BABEL_2_1_STABLE__PG_14_3
   extension_branch: BABEL_2_1_STABLE
+'14.5':
+  engine_branch: latest
+  extension_branch: latest  
```

Comment:
`latest` is okay for now but we need to change these branches to stable once they are cut.